modules/eks/actions-runner-controller/default.auto.tfvars

```diff
@@ -0,0 +1,24 @@
+enabled = false
```

Comment:
Delete `default.auto.tfvars` (Cloud Posse policy going forward is to remove all `auto.tfvars` files). Make sure everything is set in stack file and README.